content/blog/2023/2023-04-23-Java-9-module-system.md

```diff
@@ -0,0 +1,169 @@
+# Java 9 Module System
```

Comment:
Please copy a header from one of the other articles and edit it so it fits to your article.

Also, please add yourself to the `data.authors.yml` file ... this should fix the automatic preview (see the link in the "conversation" tab of the pull request).